system_files/shared/usr/libexec/configure-grub.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source /usr/lib/ujust/ujust.sh
+GRUB_STATE="$(sudo grub2-editenv list | grep "^menu_auto_hide=")"
+if [ "$GRUB_STATE" == "menu_auto_hide=0" ] ; then
```

Comment:
This isn't set on older installs. If the variable doesn't exist it is also not hidden.

Reply:
it should exist since Fedora 29, did bluefin remove those env variable?

Reply:
They weren't on my original 37 and 38 silverblue installs. Unsure why.

But it's besides the point if the variable doesn't exist it's not hidden which is what your code currently says.